ambry-router/src/test/java/com/github/ambry/router/RouterTestHelpers.java

```diff
@@ -198,13 +197,14 @@ static void setServerErrorCodes(ServerErrorCode[] serverErrorCodesInOrder, Parti
   /**
    * Asserts that expected threads are not running after the router is closed.
    */
-  static void assertCloseCleanup(NonBlockingRouter router) {
```

Comment:
I may prefer we keep the original assertCloseCleanup function without the operationCount since zero is the ideal case.
We just add the new one with the operationCount.

Reply:
I have different opinion. We should not by default assume the operation count before test is 0 if we really care about the isolation of different unit tests. We want to make sure there is no unfinished operation created by the current unit test, then the correct way of doing it, is to compare the before and after operation counts.
Even if we fixed all the unit tests that break the operation count, we still would prefer having a before operation count to compare with when we clean up for the current unit test.